### ISSUE TITLE
Flatpak Verion has no access to network shares

### ISSUE THREAD

**Issue by Slomo17** [gnome-mpv/gnome-mpv#396]
Overview Description:
Flatpak Verion has no access to files on network shares
Trying to play a file on a network share like samba or ssh fails with an error message
Steps to Reproduce:
1. Open Network share (samba or sftp)
2. try to play file with gnome-mpv

Actual Results:
Error message: Playback was terminated abnormally. Reason: loading failed.

Expected Results:
Gnome-mpv should be able to access network shares

Version:
0.15

Additional Information:
only flatpak version seems to be affected, probably because of the sandbox. Gnome-mpv installed from other sources works fine

**Comment by TingPing** [comment]
Normally that probably works over GVFS but `mpv` doesn't use that. It is probably worth opening an issue on https://github.com/flatpak/xdg-desktop-portal saying that selecting a file on an sftp mount does not work.

**Comment by saivert** [comment]
Solved with `flatpak override --filesystem=xdg-run/gvfs`
`--filesystem=host` does not include any of the stuff in the `xdg-run` directory.

**Comment by aindriu80** [comment]
I just upgrade from Ubuntu 20.04 to 20.10 and get the same error:  

> Playback was terminated abnormally. Reason: loading failed.

I tried `sudo flatpak override --filesystem=xdg-run/gvfs io.github.GnomeMpv` but 
can't play videos over LAN.  Any fix ? I tried both Flatpak and apt repository

**Comment by gnome-mpv** [comment]
@aindriu80: Try running the command again with the current app ID: `sudo flatpak override --filesystem=xdg-run/gvfs io.github.celluloid_player.Celluloid`.

**Comment by aindriu80** [comment]
> @aindriu80: Try running the command again with the current app ID: `sudo flatpak override --filesystem=xdg-run/gvfs io.github.celluloid_player.Celluloid`.

no luck. I've tried reinstalled it with:
`sudo add-apt-repository ppa:xuzhen666/gnome-mpv
sudo apt update
sudo apt install celluloid`

**Comment by aplatypus** [comment]
Yes i found this on ubuntu 21.10 today too. It is only the flatpak version.  It worked with the PPA install yesterday.   Be fort i installed the flatpak version
